docs/snippets/build-func-intro.md

```diff
@@ -0,0 +1,5 @@
+<!-- Snippet used in the following topics:
+- /docs/getting-started/build-run-deploy-func.md
+- /docs/functions/build-run-deploy-func.md
+-->
+Building a function creates an OCI container image for your function that is pushed to a container registry. It does not run or deploy the function, which can be useful if you want to build a container image for your function locally, but do not want to automatically run the function or deploy it to a cluster, for example, in a testing scenario.
```

Comment:
Ac
```suggestion
Building a function creates an OCI container image for your function that can be pushed to a container registry. It does not run or deploy the function, which can be useful if you want to build a container image for your function locally, but do not want to automatically run the function or deploy it to a cluster, for example, in a testing scenario.
```
The image is not pushed to a container registry during the build step, but rather during deploy. Users can, however, run `func build` and manually push the image to a registry if they desire.